src/github.rs

```diff
@@ -40,15 +47,16 @@ impl ServiceFactory for GithubServiceFactory {
 
     fn create_service(&self, matches: &getopts::Matches) -> CreateServiceResult {
         match (
+            matches.opt_str("github-endpoint"),
             matches.opt_str("github-org"),
             matches.opt_str("github-username"),
             matches.opt_str("github-password"),
         ) {
-            (Some(org), Some(username), Some(password)) => CreateServiceResult::Service(Box::new(
-                GithubService{org: org, username: username, password: password}
+            (endpoint, Some(org), Some(username), Some(password)) => CreateServiceResult::Service(Box::new(
+                GithubService{endpoint: endpoint.unwrap_or(DEFAULT_ENDPOINT.to_string()), org: org, username: username, password: password}
```

Comment:
Do you have a preference where the curly brace goes? Own line or on a line with the parens?